VimeoNetworking/Sources/VimeoClient.swift

```diff
@@ -131,7 +131,7 @@ final public class VimeoClient
     // MARK: - Authentication
     
         /// Stores the current account, if one exists
-    public internal(set) var currentAccount: VIMAccount?
+    public var currentAccount: VIMAccount?
```

Comment:
Ahh this is my mistake. I initially removed it while running some tests. I need to extract a call in this `didSet` so that I can properly run the unit test.